### PR TITLE
Misc fixes and enable tests on appveyor

### PR DESCRIPTION
**Pull request by eminence**
The first commit removes an unused import, which reduces the amount of warning spam on build.

The second commit was needed to get the tests to build on my windows machine (which uses back-slashes in paths).

### CODE REVIEW

**Comment by eminence** [comment]
Opps.  Turns out that import was actually used.  Trying again with an `allow(unused_imports)` instead.

**Comment by alexcrichton** [comment]
Thanks! Could some tests be added as well?

**Comment by eminence** [comment]
I didn't add any new features, but sure.  What did you have in mind?

**Comment by ohanar** [comment]
The appveyor script is pretty minimal right now, it would be good to have at least the `js-sys` and `webidl-tests` crates tested on there.

**Comment by alexcrichton** [comment]
@eminence oh primarily just tests to ensure that the behavior doesn't regress accidentally in the future through a refactor accidentally gone wrong. 

For example I'm not sure what generates the unused imports, but having a windows-specific tests with a Windows-specific path would be good to have!

**Comment by eminence** [comment]
Ok!  So I don't know how to add a test for unused imports (which wasn't a problem, just a small annoyance), but I took @ohanar's suggestion and enabled from more tests on Appveyor.

Appveyor already includes geckodriver and chromedriver as pre-installed software on its image, so this was pretty easy.  Unfortunately, I wasn't able to get geckodriver to work, so this PR only uses chromedriver.  (Any experts can review the geckodriver failure logs [here](https://ci.appveyor.com/project/eminence/wasm-bindgen/build/1.0.4)).

File: .appveyor.yml [hunk 1]
```diff
@@ -22,6 +22,13 @@ test_script:
   - npm ci
   - cargo test
   - cargo build --release -p wasm-bindgen-cli
+  - where chromedriver
+  - set CHROMEDRIVER=C:\Tools\WebDriver\chromedriver.exe
+  - cargo test -p js-sys --target wasm32-unknown-unknown
+  - cargo test -p web-sys
+  - cargo test -p web-sys --target wasm32-unknown-unknown
+  - cargo test -p webidl-tests
```
Comment (ohanar) [inline .appveyor.yml]: The web-sys and webidl-tests on the native platform are unnecessary, and just take extra time building. These crates only have tests for `wasm32-unknown-unknown`.

**Comment by alexcrichton** [comment]
@eminence oh I think I just ran across the `GlobalStack` unused import issue, which is a legit but I think we need to fix. In that sense could the unused import be fixed instead of ignored?

File: .appveyor.yml (hunk 1 above)
Comment (eminence) [reply .appveyor.yml]: Great.  I suspected that was the case, but thanks for confirming!

**Comment by alexcrichton** [comment]
Looks great, thanks!

**Pull request merged**